system-test/kitchen.ts

```diff
@@ -25,7 +25,7 @@ describe('end to end', () => {
         .on('error', done)
         .on('response',
             (resp) => {
-              uploadSucceeded = resp.statusCode === 200;
+              uploadSucceeded = resp.status === 200;
```

Comment:
So technically, is this an API breaking change?

Reply:
Yeah, I believe it's safest to call it that. Adding a reminder to update the readme which tells users what `response` emits.